api_types.ts

```diff
@@ -1107,6 +1139,99 @@ export enum PrecannedDateRange {
   Everything = 'everything',
 }
 
+/**
+ * The set of date ranges whose end dates are today.
+ */
+export const PastRelativeDateRanges = [
```

Comment:
Trying to make sure I understand the vocabulary. The existing vocabulary and comments next to precanned date ranges describe them as "live", meaning "require evaluation at runtime with regards to the current date", which I think is a useful term. When you say "relative" in these places do you mean "live" or do you mean something new like "live and has today as the start or end of the range"?

Reply:
Ah, thanks, I was using it for both in different places, inconsistently. For most of these, I will just switch to using "Live" instead of "Relative", but I'm not sure what to do about "PastRelativeDateRanges" and "FutureRelativeDateRanges" (which are the "live and has today as start/end" meaning), because "Relative" doesn't seem obvious enough after sleeping on it. Brainstorm: PastDateRangesUntilNow, PastToNowDateRanges, ...?

cc @ekoleda-codaio

Reply:
I changed these to "UntilNowDateRanges" and "FromNowDateRanges"